input/data/2009/2009A%26A...499..723A/tev-000132.yaml

```diff
@@ -1,4 +1,26 @@
 source_id: 132
 reference_id: 2009A&A...499..723A
 
-# TODO: add data
+data:
+  # TODO: No livetime found in the paper
+  livetime:
+  significance: 12.1
+  excess: 689
+
+pos:
+  ra: {val: 40d23m9.2s, err: 2.4m, err_sys: 20s}
+  dec: {val: 0d47m10.1s, err: 2.4m, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  # TODO: sigma and sigma2 have assymmetric errors: -0.03, +0.04, no 'pa' value found
```

Comment:
Usually when values aren't given, the idea is to just not put the key at all in the input files.
So no default needed for `pa` and `sigma2` here, just omit it completely in the input file.

OK?